tools/xdp/xtask/src/ci.rs

```diff
@@ -0,0 +1,41 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{build_ebpf, disasm, run};
+use anyhow::anyhow;
+use std::io;
+
+pub fn run() -> Result<(), anyhow::Error> {
+    let before = dump()?;
```

Comment:
Hmm, where is the before build from? Are we assuming the build should always be identical to the LFS files committed in this PR?

Reply:
Yeah this check is to ensure the LFS build is in sync with what the code says. I didn't want to diff the actual binary since there can be some nondeterminism so we're just diffing the disassembly output, which should be good enough for this purpose.